code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -10,73 +10,106 @@
+	It served as the testbed of several breakthroughs in laser weapon research that wouldn't trickle into mainstream laser weapons for decades - whilst modern laserarms have a similar performance, \
+	it was vastly superior to any of its contemporaries back when it was originally made.<br><br>\
+	The most revolutionary component is the miniaturised nuclear fission reactor that charges the gun's power cell, allowing for theoretically unlimited shots. \
+	All radiation emitted is contained by a plasma-lead alloy, making it safe to handle. \
+	This single example cost a considerable fortune to produce, and has never been successfully replicated. \
+	The gun's furniture and decorative embellishments were all hand-made by small group highly trained artisans for a comparatively tiny cost.<br><br>\
+	The laser was passed down several generations until it finally ended up in Epsilon Eridani. Today, it is kept on this station as a memento and reminder of what Nanotrasen R&D can achieve. \
+	Its age, craftsmanship, and unique technology marks it as one of the most expensive objects in the entire station that can be stuffed into a backpack-sized container."
 
 /obj/item/gun/energy/laser/captain/Initialize(mapload, ...)
 	. = ..()
 	RegisterSignal(src, COMSIG_PARENT_QDELETING, PROC_REF(alert_admins_on_destroy))
 
 /obj/item/gun/energy/laser/captain/examine(mob/user)
 	. = ..()
-	. += "<span class='notice'>The power cell of this item slowly recharges on it's own. This caused the station pictured to explode.</span>"
+	. += "<span class='notice'>The power cell of this item slowly recharges on it's own.</span>"
 
 /obj/item/gun/energy/laser/captain/scattershot
 	name = "scatter shot laser rifle"
 	icon_state = "lasercannon"
 	item_state = "laser"
-	desc = "An industrial-grade heavy-duty laser rifle with a modified laser lense to scatter its shot into multiple smaller lasers. The inner-core can self-charge for theorically infinite use."
+	desc = "A heavy-duty laser rifle fitted with a retractable prism that scatters its beam into multiple smaller shots. The fire selector has two settings: 'scatter', and 'kill'."
 	origin_tech = "combat=5;materials=4;powerstorage=4"
 	ammo_type = list(/obj/item/ammo_casing/energy/laser/scatter, /obj/item/ammo_casing/energy/laser)
 	shaded_charge = FALSE
-
-/obj/item/gun/energy/laser/cyborg
-	can_charge = FALSE
-	desc = "An energy-based laser gun that draws power from the cyborg's internal energy cell directly. So this is what freedom looks like?"
-	ammo_type = list(/obj/item/ammo_casing/energy/laser/cyborg)
-	origin_tech = null
-
-/obj/item/gun/energy/laser/cyborg/newshot()
-	..()
-	robocharge()
-
-/obj/item/gun/energy/laser/cyborg/emp_act()
-	return
+	extended_description = "<span class='notice'><i>You examine the scatter shot laser rifle closer, but find nothing of interest...</i></span>"	// Never seen this get used so it probably doesn't need an extended description.
```

Comment:
just make this proc's `examine_more()` return `list()` to ensure it doesn't capture the parent's version.

Reply:
It is done.